zap/src/main/java/org/parosproxy/paros/core/scanner/Variant.java

```diff
@@ -102,4 +102,6 @@ default String getLeafName(String nodeName, HttpMessage msg) {
     default List<String> getTreePath(HttpMessage msg) throws URIException {
         return null;
     }
+
+    default void decodeResponseBody(HttpMessage msg) {}
```

Comment:
Needs JavaDoc and since tag.

Reply:
It does, I'd also add, "... given {@code HttpMessage}, for example, to change a binary payload to clear text to allow to do text/string comparisons of the content."